plugins/in_tail/tail_db.c

```diff
@@ -116,11 +141,26 @@ static int db_file_exists(struct flb_tail_file *file,
         /* id: column 0 */
         *id = sqlite3_column_int64(ctx->stmt_get_file, 0);
 
+        /* name: column 1 */
+        name = sqlite3_column_text(ctx->stmt_get_file, 1);
```

Comment:
the return value for this call should be checked for a NULL value, just in case.